default/stringtables/en.txt

```diff
@@ -2579,6 +2579,9 @@ The Save Game button is disabled, either because this is a multiplayer game in w
 GAME_MENU_REALLY_CONCEDE
 Confirm conceding? This empire will be removed from the game.
 
+GAME_MENU_CONFIRM_NOT_READY
+Confirm quit in unready state?
+
```

Comment:
I would maybe suggest rewording this.  "Unready" seems a little ambiguous to me.

How would you feel about "Confirm quit?  Orders are not finalized."  Or something like that?

Reply:
As far as I am aware you don't need to click the turn button to finalize or submit your orders, that happens automatically. So that make the purpose of clicking the button to indicate to the server that you are ready for the turn to advance so that it can proceed when all the players are ready without waiting for the timer to expire.

That said I don't think my wording is ideal, I was just trying to be concise. The name of that feature is not really given in the game. Maybe the closest thing is "waiting" which is used in the code for the enum and the text in the messages box "Waiting for Other Players".

Some other possible options:
"Confirm quit? You are not waiting."
"Confirm quit? You are not waiting for turn advance."
"Confirm quit? Waiting status not set."
"Confirm quit? Ready status not set."
"Confirm quit without setting ready status?"

Reply:
Oh, i thought you did need to click the turn button for your orders to process, but that was just an assumption on my part.  My mistake.

Maybe just something to indicate what the player isn't ready for?
"Confirm quit?  Still revising orders."
"Confirm quit?  Turn advance not set."
"Confirm quit?  Still revising orders, turn advance not set."
"Confirm quit without setting readiness for next turn?"

Reply:
I like "Confirm quit? Still revising orders." That matches up with how the button changes when you click it which would help indicate what the user needs to do.

Reply:
I'm happy with that.

Reply:
String has been updated.